e2e/interchain_accounts_test.go

```diff
@@ -208,18 +248,181 @@ func (s *InterchainAccountsTestSuite) TestMsgSubmitTx_FailedTransfer_Insufficien
+
+	chainARelayerUser, chainBRelayerUser := s.GetRelayerUsers(ctx)
+	relayerAStartingBalance, err := s.GetChainANativeBalance(ctx, chainARelayerUser)
+	s.Require().NoError(err)
+	t.Logf("relayer A user starting with balance: %d", relayerAStartingBalance)
+
+	controllerAccount := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	chainBAccount := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
+
+	t.Run("register interchain account", func(t *testing.T) {
+		version := "" // allow app to handle the version as appropriate.
+		msgRegisterAccount := intertxtypes.NewMsgRegisterAccount(controllerAccount.Bech32Address(chainA.Config().Bech32Prefix), ibctesting.FirstConnectionID, version)
+		err := s.RegisterInterchainAccount(ctx, chainA, controllerAccount, msgRegisterAccount)
+		s.Require().NoError(err)
+	})
+
+	t.Run("start relayer", func(t *testing.T) {
+		s.StartRelayer(relayer)
+	})
+
+	var channelOutput ibc.ChannelOutput
+	t.Run("verify interchain account", func(t *testing.T) {
+		var err error
+		interchainAcc, err = s.QueryInterchainAccount(ctx, chainA, controllerAccount.Bech32Address(chainA.Config().Bech32Prefix), ibctesting.FirstConnectionID)
+		s.Require().NoError(err)
+		s.Require().NotZero(len(interchainAcc))
+
+		channels, err := relayer.GetChannels(ctx, s.GetRelayerExecReporter(), chainA.Config().ChainID)
+		s.Require().NoError(err)
+		s.Require().Equal(len(channels), 2)
+
+		// interchain accounts channel at index: 0
+		channelOutput = channels[0]
+	})
+
+	t.Run("execute interchain account bank send through controller", func(t *testing.T) {
+		t.Run("fund interchain account wallet on host chainB", func(t *testing.T) {
+			// fund the interchain account so it has some $$ to send
+			err := chainB.SendFunds(ctx, ibctest.FaucetAccountKeyName, ibc.WalletAmount{
+				Address: interchainAcc,
+				Amount:  testvalues.StartingTokenAmount,
+				Denom:   chainB.Config().Denom,
+			})
+			s.Require().NoError(err)
+		})
+
+		t.Run("register counterparty payee", func(t *testing.T) {
+			resp, err := s.RegisterCounterPartyPayee(ctx, chainB, chainBRelayerUser, channelOutput.Counterparty.PortID, channelOutput.Counterparty.ChannelID, chainBRelayerWallet.Address, chainARelayerWallet.Address)
+			s.Require().NoError(err)
+			s.AssertValidTxResponse(resp)
+		})
+
+		t.Run("verify counterparty payee", func(t *testing.T) {
+			address, err := s.QueryCounterPartyPayee(ctx, chainB, chainBRelayerWallet.Address, channelOutput.Counterparty.ChannelID)
+			s.Require().NoError(err)
+			s.Require().Equal(chainARelayerWallet.Address, address)
+		})
+
+		t.Run("no incentivized packets", func(t *testing.T) {
+			packets, err := s.QueryIncentivizedPacketsForChannel(ctx, chainA, channelOutput.PortID, channelOutput.ChannelID)
+			s.Require().NoError(err)
+			s.Require().Empty(packets)
+		})
+
+		t.Run("stop relayer", func(t *testing.T) {
+			err := relayer.StopRelayer(ctx, s.GetRelayerExecReporter())
+			s.Require().NoError(err)
```

Comment:
for simplicity, I wonder if we should just make this a suite function so it matches `s.StartRelayer(relayer)`